src/components/oppfolgingsoppgave/OppfolgingsoppgaveModal.tsx

```diff
@@ -96,27 +96,48 @@ export const OppfolgingsoppgaveModal = ({
   toggleOpen,
   existingOppfolgingsoppgave,
 }: Props) => {
-  const [isGeneralFormError, setIsGeneralFormError] = useState(false);
+  const [isFormError, setIsFormError] = useState(false);
+  const [isFristTouched, setIsFristTouched] = useState(false);
   const createOppfolgingsoppgave = useCreateOppfolgingsoppgave();
   const editOppfolgingsoppgave = useEditOppfolgingsoppgave(
     existingOppfolgingsoppgave?.uuid
   );
   const {
     register,
-    formState: { errors },
+    formState: { errors, touchedFields },
     handleSubmit,
     setValue,
     watch,
-  } = useForm<FormValues>();
+  } = useForm<FormValues>({
+    defaultValues: {
+      beskrivelse: existingOppfolgingsoppgave?.tekst ?? "",
+      frist: existingOppfolgingsoppgave?.frist ?? null,
+    },
+  });
+  const watchedValues = watch();
   const isEditMode = !!existingOppfolgingsoppgave;
 
+  const isFormEdited = useCallback(
+    (existingOppfolgingsoppgave) => {
+      const isFristEdited =
+        watchedValues.frist !== existingOppfolgingsoppgave?.frist;
+      const isBeskrivelseEdited =
+        watchedValues.beskrivelse !== existingOppfolgingsoppgave?.tekst;
+      return isFristEdited || isBeskrivelseEdited;
+    },
+    [watchedValues]
+  );
+
+  useEffect(() => {
+    if (touchedFields["beskrivelse"] || isFristTouched) {
+      setIsFormError(!isFormEdited(existingOppfolgingsoppgave));
+    }
+  }, [existingOppfolgingsoppgave, isFormEdited, isFristTouched, touchedFields]);
+
```

Comment:
Kan dette gjøres på en enklere måte? 🤔
Dette er der nå siden vi må sette form error etter `onBlur` av noen av feltene i skjemaet. Man bør også oppdatere error når noen av feltene endrer seg.

Reply:
Har du sjekket https://react-hook-form.com/docs/useform ? Er noe `mode` / `reValidateMode` som styrer når det valideres.
Føler dette burde kunne løses med rammeverket :thinking:

Reply:
Det er litt denne dato komponenten som skaper problemer her 🥲 Prøver litt til for å se om react hook form har noe lurt